### ISSUE TITLE
About symmetrically normalization of adjacency matrix

### ISSUE THREAD

**Issue by empty-id** [tkipf/pygcn#55]
Hi, I notice that in this PyTorch version code, the adjacency matrix is row-normalized instead of symmetrically normalized. However, the accuracy (82.5%) is higher than the TensorFlow version code (81.6%). Moreover, I also tried to symmetrically normalize the adjacency matrix in this PyTorch version, but the result dropped (to 79.9%). Nevertheless, result of TensorFlow version does not change after modification of normalization. For summarization, this is the experiments I did:

| Cora dataset   |  TensorFlow   |  PyTorch   |
| :-: | :-: | :-: |
|   Symmetrically Normalization  |   81.6  |  79.9   |
|    Row Normalization |  81.6   |   82.5  |

Is there any idea why does this happen?

**Comment by empty-id** [comment]
I find some differences between pytorch and tf version.

* In layers.py, pytorch version should use glorot initialization for weights and zero initialization for bias
* In models.py, pytorch doesn't perform dropout (or sparse_dropout) for the input x first.

After I modified the above two points, the result now is ~82% for symmetrically normalization in pytorch version.
